vNext/extensions/applicationinsights-react-native/package.json

```diff
@@ -35,8 +35,7 @@
         "tslib": "^1.9.3"
     },
     "peerDependencies": {
-        "react-native": "*",
-        "react-native-device-info": "^0.26.4"
```

Comment:
you dont want this anymore?

Reply:
I think I only want react-native as a peer. For device-info, users will need to link it anyways, so this is just so it shows up on the npmjs dependency tree. So I moved from peer to pure dependency